gke-disk-image-builder/cli/main.go

```diff
@@ -37,10 +38,15 @@ func (s *stringSlice) Set(value string) error {
 	return nil
 }
 
+var (
+	gcpResourceNameRegex = regexp.MustCompile("^[a-z]([-a-z0-9]*[a-z0-9])?$")
+)
+
 func main() {
 	var containerImages stringSlice
 	projectName := flag.String("project-name", "", "name of a gcp project where the script will be run")
 	imageName := flag.String("image-name", "", "name of the image that will be generated")
+	imageFamilyName := flag.String("image-family-name", "secondary-disk-image", "name of the image family associated with the created disk image")
```

Comment:
Is "secondary-disk-image" a valid image family name? I haven't heard of it, if it's not let's not set it as a default.

Reply:
It is the image family name currently hardcoded in the disk image builder code, so I'm using it as the default. Shall I change to something else?

Reply:
Oh weird, I never noticed that, ok sounds good.